### PR TITLE
initialize project with components

### PR DESCRIPTION
**Pull request by rega1237**
## Learning objectives
- Use React Router.
- Use React components.

## Description
This project will lay foundations for my Bookstore website. I created the React part of the app. I also set up routing using React Router.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <Route path="/" exact>
+          <BooksPage />
+        </Route>
+        <Route path="/categories">
+          <CategoriesPage />
+        </Route>
+      </Switch>
+    </div>
+  );
+}
```
Comment (elmar8287) [inline src/App.js:21]: - [ ] Please, consider creating and adding the component for creating the new book, which renders a form with inputs for the title and the author of the book. It should also have a button to submit, which wouldn't add the book yet, this is just a placeholder for now 👍

File: src/components/Categories/Categories.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <section>
+    <h1>CATEGORIES</h1>
```
Comment (elmar8287) [inline src/components/Categories/Categories.js]: - [ ] Please, consider here should be display only a button with text "Check status"

File: README.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
```
Comment (elmar8287) [inline README.md]: - [ ] Please, consider that this README file came with CRA (create-react-app), so you have to replace it with [MICROVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues
- Good PR description

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @rega1237 , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Descriptive readme file.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "bug fix".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**